build.sbt

```diff
@@ -11,6 +11,7 @@ lazy val commonsService = nexusDep("commons-service", commonsVersion)
 lazy val commonsTest    = nexusDep("commons-test", commonsVersion)
 lazy val shaclValidator = nexusDep("shacl-validator", commonsVersion)
 lazy val sparqlClient   = nexusDep("sparql-client", commonsVersion)
+lazy val iamTypes       = nexusDep("iam", commonsVersion)
```

Comment:
should we also rename the variable name, as it's not only types now, but also a client.